### PR TITLE
Redesigns Deltastation's white ship into a luxury NT frigate.

### PR DESCRIPTION
**Pull request by WJohn**
Delta's white ship has always come off as too large, empty, and boxy to me, personally, so I set out to redesign it into something with an actual shape and more details!

Also, it's infested with starving giant spiders. They have half as much health and damage as their regular versions so they're not a complete nightmare to take out, but they're still dangerous to an unarmed explorer.

:cl: WJohnston
add: Redesigned deltastation's abandoned white ship into a luxury NT frigate. Oh, and there are giant spiders too.
/:cl:

### CODE REVIEW

**Comment by Mark9013100** [comment]
Ten times more interesting than the current whiteship, nice.

**Comment by WJohn** [comment]
Yes, it has outfits in the locker above the bridge.

**Comment by 81Denton** [comment]
That design is really nice, I like that it comes with power and atmos.
Circuit board/defib/sunglasses/somatoray removal should probably be mentioned in the CL.

**Comment by Tlaltecuhtli** [comment]
spiders arent space proof also it needs more wood/carpet

**Comment by Bombpants** [comment]
A small botany area would be nice

**Review by ShizCalev — DISMISSED** [review]

**Comment by coiax** [comment]
Don't passive spiders lay webs when active anyway? Do they need to be mapped in?

**Comment by subject217** [comment]
Any chance you leave a mini egun mapped in there, somewhere? It was nice to see it used. Could go in the bridge.

**Comment by MrDoomBringer** [comment]
yay you added it! such a better design <3

**Comment by WJohn** [comment]
@coiax to my knowledge only nurses do, of which there's only one, and they only do it when they're active and wandering around when a living player is on the level.

**Comment by Bawhoppen** [comment]
This change is only good.

**Review by ninjanomnom — CHANGES_REQUESTED** [review]

**Comment by WJohn** [comment]
oh! my bad, i completely forgot to do that.

**Comment by ninjanomnom** [comment]
No look at the comments in that file. Your area definitions are in the single area shuttle section and this isn't a single area shuttle.

File: code/game/area/areas/shuttles.dm [hunk 1]
```diff
@@ -62,6 +62,43 @@
...
+////////////////////////////White Ship////////////////////////////
+
+/area/shuttle/abandoned
+	name = "Abandoned Ship"
+	blob_allowed = FALSE
+	canSmoothWithAreas = /area/shuttle/abandoned
+
+/area/shuttle/abandoned/bridge
+	name = "Abandoned Ship Bridge"
+	blob_allowed = FALSE
```
Comment (ninjanomnom) [inline code/game/area/areas/shuttles.dm]: blob_allowed is set to false already in `/area/shuttle/abandoned`

**Review by ninjanomnom — CHANGES_REQUESTED** [review]

**Review by ninjanomnom — APPROVED** [review]

**Pull request merged**